packages/experiments/src/components/FolderCover/FolderCover.scss

```diff
@@ -88,19 +91,19 @@
 
   .isFluent & {
     @include text-align(left);
-    color: $ms-color-yellowDark;
+    color: $folder-accent;
```

Comment:
we should make this a theme color, otherwise it'll break/be unchangeable if it ever gets used on SharePoint (and when we do theming in OneDrive, like for dark mode)

Reply:
I was thinking of the same thing so i consulted with @Jahnp on the best approach and where to declare the color value, here's what we resolved: This color value is for use on text and icons that will only ever appear on top of the folder SVG itself, never set against a theme-controlled background, so it actually shouldnt be tied to the theme, but rather only ever updated if we change out the folder plate svgs like this one: https://spoprod-a.akamaihd.net/files/fabric/office-ui-fabric-react-assets/foldericons-fluent/lg-fg.svg